addons/binding/org.openhab.binding.milight/src/main/java/org/openhab/binding/milight/internal/protocol/QueuedSend.java

```diff
@@ -179,7 +179,7 @@ private void remove_from_queue(int unique_command_id) {
                 }
             } catch (IllegalStateException e) {
                 // Ignore threading errors
-                logger.error(e.getLocalizedMessage());
+                logger.error("{}", e.getLocalizedMessage());
```

Comment:
I would log the exception instead, it is an error, I want to be able to debug the stacktrace